ingestion/functions/template.yaml

```diff
@@ -149,6 +149,17 @@ Resources:
       Policies:
         - AWSLambdaBasicExecutionRole
         - AWSLambdaReadOnlyAccess
+  BrazilRGDSParsingFunction:
+    Type: AWS::Serverless::Function # More info about Function Resource: https://github.com/awslabs/serverless-application-model/blob/master/versions/2016-10-31.md#awsserverlessfunction
+    Properties:
+      CodeUri: parsing/riograndedosul/
+      Handler: riograndedosul.lambda_handler
+      Description: Parse case data for the Rio Grande do Sul State of Brazil
+      # Function's execution role
+      Policies:
+        - AWSLambdaBasicExecutionRole
+        - AWSLambdaReadOnlyAccess
+      MemorySize: 512
```

Comment:
If you're parsing CSV line by line I'm surprized this is needed, this was needed for Japan when it's one json blob that we can't stream-parse but it your case it shouldn't be needed and will just increase the costs of running the function on aws.
On a local run, what was the max memory usage printed at the end? (sam local invoke prints that info)